_pages/develop/map.md

```diff
@@ -0,0 +1,682 @@
+**You should not use this tile service for production applications; you will
+need to implement or purchase your own tile service.**
+
+The [OpenStreetMap Tile Usage Policy](https://operations.osmfoundation.org/policies/tiles/)
+details specific restrictions for developers using their public tile service.
+These restrictions include the following provision: "Heavy use (e.g. distributing
+an app that uses tiles from openstreetmap.org) is forbidden without prior
+permission." If you choose to use the OpenStreetMap service for demos or development,
+you should likely replace the service URLs when your app is shipped to production.
+
+The following list of companies provide tile service APIs that may be free or
+paid and should be compatible with the `px-map-tile-layer` (note that these are
+not endorsed, but just offered as options):
+
+  - [Mapbox](https://www.mapbox.com/help/how-mapbox-data-works/)
+  - [Carto](https://carto.com/location-data-services/basemaps/)
+  - [Mapzen](https://mapzen.com/products/maps/)
+
+
+You can also use the `px-map-tile-layer-bing` component to load map tiles from
+the Bing API, the `<px-map-tile-layer-google>` component to load map tiles from
+the Google Maps API, or the `<px-map-tile-layer-wms>` component to display WMS
+services as tile layers on the map.
+
+## Example: creating a px-map in your application
+
+<iframe height='480' scrolling='no' title='px-map-demo'
+src='//codepen.io/talimarcus/embed/BYxyNb/?height=265&theme-id=0&default-tab=html,result&embed-version=2'
+frameborder='no' allowtransparency='true' allowfullscreen='true' style='width: 100%;'></iframe>
+
+# Geospatial Data Formats
+
+Geospatial data is available in many different formats. This can make it confusing
+to choose which data format to use in your application. Some formats are open,
+while others are proprietary to specific vendors. Some formats express data in
+plain text, while others are encoded and require specific libraries to translate
+them. One geospatial data format that has gained popularity in recent years is
+GeoJSON. GeoJSON is a very simple open format that expresses data in plain text
+and has been adopted by a wide range of vendor and tools. Lots of data sources
+are available as GeoJSON.
+
+Px-map doesn’t prescribe what data format its developers should use, but GeoJSON
+has many advantages, and there is dedicated support for it within px-map.
+
+## Understanding GeoJSON
+
+GeoJSON is a JSON-based format used to represent a variety of geographic data
+structures. You can use GeoJSON to represent geometry (e.g. points, lines, areas),
+features, or feature collections. GeoJSON is always composed of a single object,
+and that object must have a member with the name "type." The type refers to the
+type of GeoJSON object and must be one of: `"Point"`, `"MultiPoint"`, `"LineString"`,
+`"MultiLineString"`, `"Polygon"`, `"MultiPolygon"`, `"GeometryCollection"`,
```

Comment:
Probably don't need to use both quotes and backticks here.

Reply:
Going to keep it like this. I want to be specific that it needs to be defined as a string, because I don't think that's obvious.